run_topostats.py

```diff
@@ -162,19 +167,37 @@ def process_scan(
         labelled_data=grains.images["labelled_regions"],
         pixel_to_nanometre_scaling=filtered_image.pixel_to_nm_scaling,
         img_name=filtered_image.filename,
-        output_dir=output_dir,
+        output_dir=output_dir / filtered_image.filename,
     )
     grain_statistics = grainstats.calculate_stats()
 
+    # Run dnatracing
```

Comment:
Would it be wise to make the dna tracing step optional via the config file / command line options for now? I guess we're still deciding how to ultimately control the workflow, but I think in the short term there are several use cases where dna tracing is not needed.

Reply:
Should definitely make workflow flexible and happy to take guidance from users on what they would like to control.

I didn't do that in this work because `pygwytracing.py` appears to always run `dnatracing` after its extracted data using Gwyddion.